mandatory/4-tax.js

```diff
@@ -28,7 +35,7 @@ There are some Tests in this file that will help you work out if your code is wo
 To run the tests for just this one file, type `npm test -- --testPathPattern 4-tax` into your terminal
 (Reminder: You must have run `npm install` one time before this will work!)
 ===================================================
-*/
+// */
```

Comment:
again, please don't modify anything in the test section (see the line which states "TESTS - DO NOT MODIFY BELOW THIS LINE")